src/governance.rst

```diff
@@ -144,6 +144,24 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
```

Comment:
We might also require the policy to be added to the docs?

Reply:
This is rendered with the docs, right?

Reply:
Sorry I meant that the new policy may go somewhere in the docs. It might be easier to read than gh issues.

Reply:
I guess that depends on the policy and that will be part of the voting.
Like the `run_exports` one, we may want to write docs for it, but the "create a bot to close stale PRs on staged-recipes" we may not want to write docs.